docs/src/refs.bib

```diff
@@ -1,4 +1,158 @@
+                  loro {R}appresentazioni, {INDAM}, {R}ome, 1972)},
+  mrnumber      = {364419},
+  publisher     = {Academic Press, London-New York},
+  pages         = {395--413},
+  year          = {1974}
+}
+
+@Misc{Car85,
+  author        = {Carter, Roger W.},
+  title         = {Finite groups of {Lie} type. {Conjugacy} classes and complex characters},
+  zbl           = {0567.20023},
+  howpublished  = {Pure and {Applied} {Mathematics}. {A} {Wiley}-{Interscience} {Publication}. {Chichester}-{New} {York}
+                  etc.: {John} {Wiley} and {Sons}. {XII}, 544 p. £42.50 (1985).},
+  year          = {1985},
+  language      = {English},
+  zbmath        = {3904833}
+}
+
+@Article{Cha68,
+  author        = {Chang, Bomshik},
+  title         = {The conjugate classes of {Chevalley} groups of type $G_2$},
+  zbl           = {0285.20043},
+  journal       = {J. Algebra},
+  fjournal      = {Journal of Algebra},
+  volume        = {9},
+  pages         = {190--211},
+  year          = {1968},
+  doi           = {10.1016/0021-8693(68)90020-3},
+  language      = {English},
+  zbmath        = {3447168}
+}
+
+@Article{DM87,
+  author        = {Deriziotis, D. I. and Michler, G. O.},
+  title         = {Character table and blocks of finite simple triality groups ${}^3D_4(q)$},
+  zbl           = {0628.20014},
+  journal       = {Trans. Am. Math. Soc.},
+  fjournal      = {Transactions of the American Mathematical Society},
+  volume        = {303},
+  pages         = {39--70},
+  year          = {1987},
+  doi           = {10.2307/2000778},
+  language      = {English},
+  zbmath        = {4021426}
+}
+
+@Article{DM87*1,
+  author        = {François Digne and Jean Michel},
+  title         = {Foncteurs de Lusztig et caractères des groupes linéaires et unitaires sur un corps fini},
+  journal       = {Journal of Algebra},
+  volume        = {107},
+  number        = {1},
+  pages         = {217--255},
+  year          = {1987},
+  doi           = {10.1016/0021-8693(87)90087-1},
+  url           = {https://www.sciencedirect.com/science/article/pii/0021869387900871}
+}
+
+@Article{EY86,
+  author        = {Enomoto, Hikoe and Yamada, Hiromichi},
+  title         = {The characters of $G_2(2^n)$},
+  zbl           = {0614.20028},
+  journal       = {Jpn. J. Math., New Ser.},
+  fjournal      = {Japanese Journal of Mathematics. New Series},
+  volume        = {12},
+  pages         = {325--377},
+  year          = {1986},
+  language      = {English},
+  zbmath        = {3993856}
+}
+
+@Article{Enn62,
+  author        = {Ennola, Veikko},
+  title         = {On the conjugacy classes of the finite unitary groups},
+  zbl           = {0105.02403},
+  journal       = {Ann. Acad. Sci. Fenn., Ser. A I},
+  fjournal      = {Annales Academiae Scientiarum Fennicae. Series A I},
+  volume        = {313},
+  pages         = {13},
+  year          = {1962},
+  language      = {English},
+  zbmath        = {3171183}
```

Comment:
no doi, check mathscinet?

Reply:
also for the others without doi

Reply:
Maybe this `10.5186/aasfm.1962.313`? I couldn't find a doi for all papers.